packages/react/src/components/CardEditor/CardEditForm/CardEditForm.test.jsx

```diff
@@ -1,55 +1,18 @@
 import React from 'react';
-import { fireEvent, render, screen } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 
 import { CARD_DIMENSIONS, CARD_SIZES, CARD_TYPES } from '../../../constants/LayoutConstants';
 
-import CardEditForm, {
-  getCardSizeText,
-  handleSubmit,
-  hideCardPropertiesForEditor,
-} from './CardEditForm';
+import CardEditForm, { getCardSizeText } from './CardEditForm';
 
-const mockSetError = jest.fn();
 const mockOnChange = jest.fn();
 const mockOnCardJsonPreview = jest.fn();
-const mockSetShowEditor = jest.fn();
-const mockOnValidateCardJson = jest.fn().mockImplementation(() => []);
-
-const cardConfig = {
-  title: 'timeSeries',
-  size: 'MEDIUM',
-  type: 'TIMESERIES',
-  content: {
-    series: [
-      {
-        dataItemId: 'torque',
-        dataSourceId: 'torque_id',
-        label: 'Torque',
-      },
-    ],
-  },
-};
 
 afterEach(() => {
   jest.clearAllMocks();
 });
 describe('CardEditForm', () => {
   describe('CardEditForm', () => {
-    it('should fire onCardJsonPreview when opening the json modal', () => {
```

Comment:
Why are we deleting these test? None of these changes have to do with the listed issues, unless I am missing something.

Reply:
As the methods are been moved from CardEditForm.jsx to CardEditor.jsx for issue https://github.ibm.com/wiotp/Maximo-Asset-Monitor/issues/2711, we had to move test cases as well to CardEditor.test.jsx

Reply:
This isn't mentioned at all in your description of the PR and there are no issues opened here explaining what is requiring this change.